sql/moz-fx-data-shared-prod/mozilla_vpn_derived/fxa_attribution_v1/query.sql

```diff
@@ -51,18 +51,41 @@ flows AS (
     fxa_content_auth_stdout_events
   WHERE
     IF(@date IS NULL, partition_date < CURRENT_DATE, partition_date = @date)
```

Comment:
I presumed that this shouldn't be limited to >= 2021-08-25, because that would exclude the first year from when VPN launched with wave 1 in July 2020.